python2/test_typing.py

```diff
@@ -1228,6 +1228,9 @@ def test_list(self):
     def test_deque(self):
         self.assertIsSubclass(collections.deque, typing.Deque)
 
+    def test_counter(self):
+        self.assertIsSubclass(collections.Counter, typing.Counter)
+
```

Comment:
Please add a test checking that a subclass of counter _could_ be instantiated, as you do for ``ChainMap`` in ``test_chainmap_subclass``. (And also for Python 3)